client/src/Home.jsx

```diff
@@ -0,0 +1,235 @@
+            </div>
+            <div className="row">
+              <h2>{greeting}</h2>
+            </div>
+            <div className="row slider">
+              {this.state.recentlyPlayed.items.map(song =>
+                <div key={song.id} className='col-third margin10 align-center pointer'>
+                  <div className='col-three-fourth'>
+                    <img className='coverart' src={song.track.album.images[0].url} alt="" />
+                  </div>
+                  <div className='col-fourth text-align-center'>
+                    <p className='song-name'>{song.track.name}</p>
+                  </div>
+                </div>)}
+            </div>
+            <div className="row">
+              <h3>{`${this.state.profile.display_name}'s`} Playlists</h3>
+            </div>
+            <div className="row slider">
+              {this.state.userPlaylists.items.map(playlist =>
+                <div key={playlist.id} className='col-third margin10 align-center pointer'>
+                  <div className='col-three-fourth'>
+                    <img className='coverart' src={playlist.images[0].url} alt="" />
+                  </div>
+                  <div className='col-fourth text-align-center'>
+                    <p className='pl-name'>{playlist.name}</p>
+                  </div>
+                </div>)}
+            </div>
+            <div className="row">
+              <h3>Featured Playlists</h3>
+            </div>
+            <div className="row slider">
+              {this.state.featuredPlaylists.playlists.items.map(playlist =>
+                <div key={playlist.id} className='col-third margin10 align-center pointer'>
+                  <div className='col-three-fourth'>
+                    <img className='coverart' src={playlist.images[0].url} alt="" />
+                  </div>
+                  <div className='col-fourth text-align-center'>
+                    <p className='pl-name'>{playlist.name}</p>
+                  </div>
+                </div>)}
+            </div>
+            <div className="row">
+              <h3>Your Followed Artists</h3>
+            </div>
+            <div className="row slider">
+              {this.state.artists.artists.items.map(artist =>
+                <div key={artist.id} className='col-third margin10 align-center'>
+                  <div className='col-three-fourth'>
+                    <img className='artists' src={artist.images[0].url} alt="" />
+                  </div>
+                  <div className='col-fourth text-align-center'>
+                    <p>{artist.name}</p>
+                  </div>
+                </div>)}
+            </div>
+          </div>
+      </div>
+
+    </>);
+  }
+}
+
+axios.defaults.baseURL = 'https://api.spotify.com/v1';
+// eslint-disable-next-line dot-notation
+axios.defaults.headers['Authorization'] = `Bearer ${window.localStorage.spotifyAccessToken}`;
+axios.defaults.headers['Content-Type'] = 'application/json';
```

Comment:
this is probably more appropriate to move into your Authorize component or App's componentDidMount or something